core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -77,8 +77,18 @@ traverse_list(bh_list *l, list_visitor visitor, void *user_data)
 }
 
 static bool
-allocate_aux_stack(WASMCluster *cluster, uint32 *start, uint32 *size)
+allocate_aux_stack(WASMExecEnv *exec_env, uint32 *start, uint32 *size)
 {
+    WASMCluster *cluster = wasm_exec_env_get_cluster(exec_env);
+#if WASM_ENABLE_HEAP_AUX_STACK_ALLOCATION == 1
+    WASMModuleInstanceCommon *module_inst =
+        wasm_exec_env_get_module_inst(exec_env);
+
+    *start = wasm_runtime_module_malloc(module_inst, cluster->stack_size, NULL);
```

Comment:
i'm not sure what's the intention of this.
for wasi-threads, we will disable this allocation, right?

Reply:
Yes, it will be disabled for wasi threads, but even with that, we need to keep `wasm_cluster_spawn_exec_env()` working; aux stack allocation is needed for that.

Reply:
ok